webui/cypress/cypress/support/index.js

```diff
@@ -14,12 +14,14 @@
 // ***********************************************************
 
 import './commands';
-import { STORAGE_KEY_AUTH } from '../constants';
+// import { STORAGE_KEY_AUTH } from '../constants';
 
-const _clear = Cypress.LocalStorage.clear;
-Cypress.LocalStorage.clear = function(aKeys) {
-  const keysToKeep = new Set([ STORAGE_KEY_AUTH ]);
-  const keys = (aKeys && aKeys.length ? aKeys : Object.keys(window.localStorage))
-    .filter(key => !keysToKeep.has(key));
-  return _clear(keys);
-};
+// const _clear = Cypress.LocalStorage.clear;
+// Cypress.LocalStorage.clear = function(aKeys) {
+//   if (aKeys && aKeys.length) return _clear.apply(Cypress.LocalStorage, arguments);
+//   const keysToKeep = new Set([ STORAGE_KEY_AUTH ]);
+//   const keys = (aKeys && aKeys.length ? aKeys : Object.keys(window.localStorage))
+//     .filter(key => !keysToKeep.has(key));
+//   const args = [ keys, Array.from(arguments).slice(1) ];
+//   return _clear.apply(Cypress.LocalStorage, args);
+// };
```

Comment:
Philosophically I found this to be confusing, because reseting the local storage cache between each test sort of limits each test to be a unit test instead of a true end-to-end.

Reply:
yeah, I think their idea is that you'd include all the "steps" for creating that end-to-end behavior in one `it` block? 

Cypress has always had this behavior of resetting the state between tests and we're just following it here so that we're use it closer to how it's designed to be used (I think).